test/Iden3Helpers.test.js

```diff
@@ -0,0 +1,217 @@
+/* global artifacts */
```

Comment:
This is outdated (mimc7), where did this come from?

Reply:
See comments above